recipes/pyfolio/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pyfolio" %}
+{% set version = "0.8.0" %}
+{% set sha256 = "e30ee8c1932b45c4ce066cc78aee957d939e02a9427e797e2f7cd582cb0938c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - ipython >=3.2.3
+    - matplotlib >=1.4.0
+    - numpy >=1.9.1
+    - pandas >=0.19.0
+    - pytz >=2014.10
+    - scipy >=0.14.0
+    - scikit-learn >=0.18.2
+    - seaborn >=0.7.1
+    - pandas-datareader >=0.2
+    - empyrical >=0.3.2
+    - pymc3 >=3.1
+    - theano
```

Comment:
Is `theano` actually used by `pyfolio` directly? Not seeing any imports of it.

FWIW `pymc3` has a hard dependency on `theano`. So if it is being pulled in for `pymc3`, it need not be.

Reply:
You are right there are none; the documentation of pyfolio says to install it along pymc3, but looking at https://github.com/pymc-devs/pymc3/blob/master/requirements.txt it is required by pymc3 so we should remove it!